frontend/docs/seo-guide.md

```diff
@@ -0,0 +1,359 @@
+- Use protocol relative URLs for all other domains.
+
+_Why:_
+> If you want to know more about SSL I recommend you read the [following answer](https://security.stackexchange.com/questions/6290/how-is-it-possible-that-people-observing-an-https-connection-being-established-w/6296#6296).
+
+#### Flash
+
+Avoid **Flash** content or pages. They are not accessible on mobile phones and will be ranked lower.
+#### iFrames
+
+Ensure there are **no content in iFrames**. Frames can cause problems for search engines because they don't correspond to the conceptual model of the web. 
+
+> Learn more [here](https://support.google.com/webmasters/answer/34445?hl=en) and [here](https://developers.google.com/speed/docs/insights/rules).
+
+#### Page Speed
+
+Page speed is a measurement of how fast the content on your page load. You can evaluate your page speed with [Google's PageSpeed Insights.](https://developers.google.com/speed/pagespeed/insights/) Google has indicated page speed **is one of the signals used by its algorithm to rank pages**.
+
+You can find some of the many ways to increase your page speed: 
+
+- Use **Gzip**, a software application for file compression, to reduce the size of your CSS, HTML, and JavaScript files.
+- **Minify CSS, JavaScript, and HTML**.
+- Remove **render-blocking JavaScript**.
+- Leverage **browser caching**.
+
+_Why:_
+> Page speed is a measurement of how fast the content on your page loads. [Read more...](https://moz.com/learn/seo/page-speed)
+
+### 2.2 Content
+
+#### General
+
+Content is the most important thing in SEO. Keep in mind:
+
+- New content is important! **Keep your page active** and Google will reward it.
+- Content does **not have orthographic or serious grammatical errors**.
+- All your pages should have **at least 300 words**. If your web has less content, is it really necessary?
+- **Text/HTML ratio** is not a direct ranking factor for search engines but there are many factors related to the ratio that indicate best SEO practices and thus may indirectly result in higher search rankings. A good text to HTML ratio is about from 25 to 70 percent.
+
+#### Title
+
+This element especifies **the title of a webpage**. Title tags are displayed on search engine results pages (SERPs) as the clickable headline for a given result, and are important for usability, SEO, and social sharing.
+
+- Is about **50-60 characters long**, including spaces.
+
+- Is concise and **contain your best keywords**.
+
+- Each page should have its own **exclusive title**.
+
+- Here you can find an (snippet optimizer)[http://www.seomofo.com/snippet-optimizer.html] that allows you to see how your titles look on Google and other search results.
+
+_Why:_
+> Title tags are a major factor in helping search engines understand what your page is about, and they are the first impression many people have of your page. [Read more...](https://moz.com/learn/seo/title-tag)
+
+#### Images
+
```

Comment:
we can add some image optimizers like:
https://tinypng.com/
https://jakearchibald.github.io/svgomg/